logicrunner/handle_additional_call.go

```diff
@@ -83,15 +83,8 @@ func (h *HandleAdditionalCallFromPreviousExecutor) Present(ctx context.Context,
 	ctx = contextWithServiceData(ctx, message.ServiceData)
 
 	done, err := h.dep.WriteAccessor.Begin(ctx, flow.Pulse(ctx))
-	if err != nil { // pulse changed, send that message to next executor
-		// ensure OK response because we might catch flow cancelled
-		msg, err := payload.NewMessage(&message)
-		if err != nil {
-			return errors.Wrap(err, "failed to serialize message")
-		}
-		_, done := h.dep.Sender.SendRole(ctx, msg, insolar.DynamicRoleVirtualExecutor, message.RequestRef)
-		done()
-		return nil
+	if err != nil {
```

Comment:
Might be another error (not currently, but in future). It will be hard to debug. Might be better to write:
```
		if err != nil {
			if err == executor.ErrWriteClosed {
				return flow.ErrCancelled
			}
			return errors.Wrap(err, "failed to acquire write access")
		}
```

Reply:
addressed